### ISSUE TITLE
Packet Parser: Unknown switch: 0A41

### ISSUE THREAD

**Issue by zanzibar222** [OpenKore/openkore#3903]
having this issue and my rebellion skill not working

skill: fire dance

![image](https://github.com/user-attachments/assets/d38a65b5-87dd-4740-b489-ac9a178ed820)

**Comment by ya4ept** [comment]
```
#if PACKETVER >= 20151223
	skillscale = 0xA41,
#endif
```

```
struct PACKET_ZC_SKILL_SCALE {
	int16 PacketType;
	uint32 AID;
	int16 skill_id;
	int16 skill_lv;
	int16 x;
	int16 y;
	uint32 casttime;
} __attribute__((packed));
```

**Comment by zanzibar222** [comment]
thank you sir

**Comment by dcyxxt** [comment]
did you fix it?

**Comment by ya4ept** [comment]
no, we need logs: https://github.com/OpenKore/openkore/wiki/How-to-collect-logs%3F

**Comment by alisonrag** [comment]
skill scale is sent in some cases instead of skill use also range is got from lub file
![image](https://github.com/user-attachments/assets/4a6dbe12-9e2d-46a3-9fe3-86b18447142b)
![image](https://github.com/user-attachments/assets/abcd1676-39a2-4563-8588-0241b93abfb9)

**Comment by alcren12** [comment]
'0A41' => ['skill_use_location', 'a4 v4 V', [qw(sourceID skillID lv x y tick)]],
